src/components/Icon/Expensicons.js

```diff
@@ -64,4 +65,5 @@ export {
     Users,
     Wallet,
     SignOut,
+    Exclamation,
```

Comment:
Nit-picky (sorry 😅) but all (...most?) of these icons are in alphabetical order currently. Let's try to keep it that way for organizational purposes.

Reply:
[Newly added icons](https://github.com/studio-504/Expensify.cash/blob/a97df32c05c8901f90c0222d32c229a0e89f97aa/src/components/Icon/Expensicons.js#L33-L41) isn't alphabetical order. Also it used to be not accurately alphabetical ordered.

So, I re-ordered completely.